packages/cozy-harvest-lib/src/components/KonnectorSuccess.jsx

```diff
@@ -40,12 +39,12 @@ export class KonnectorSuccess extends Component {
   }
   render() {
     const { t } = this.props
-    const relatedApps = sortBy(
-      Object.values(KonnectorSuccess.apps).filter(app =>
-        app.predicate(this.state, this.props)
-      ),
-      app => -app.priority
-    )
+    const relatedApps = getRelatedAppsSlugs({
+      konnectorManifest: this.props.konnector,
+      trigger: this.state.trigger
+    })
+      .map(slug => KonnectorSuccess.apps[slug])
+      .filter(Boolean)
```

Comment:
We lose the notion of priority here. It was needed for our AF project. We should certainly keep it, no?

In fact, I think we have it also on the `DataTabs` but it is implicit right? We should keep it explicit

Reply:
The priority is now determined by the order in the configuration array rather than a `priority` number. I figured we could swap the array order around in our AF projects if we want a different order there. I don't mind putting the number system back in if you think this is not explicit enough.

Reply:
I would prefer yep. Explicit is way better :D.

Also can we extract the `appLinksProps` in a new file since we already now we have to override it? :D